### ISSUE TITLE
Goes straight back to login screen

### ISSUE THREAD

**Issue by Arcath** [cedrozor/myrtille#82]
My Myrtille setup stopped working in the last week.

I've updated to 1.6.1 and ensured that WebSockets are enabled in IIS.

Now I've cleared most of the errors but I'm stuck with a setup that will login and then drop you back to the login screen straight away. I don't see any sign that it made an RDP connection.

`Myrtille.Services.log` logs:

```
2018-01-17 15:15:57,139 [13] INFO  System.Diagnostics redirection [(null)] - Started remote session 3
2018-01-17 15:15:58,199 [13] INFO  System.Diagnostics redirection [(null)] - Stopped remote session 3
```

I don't get anything logged into `Myrtille.Web.log`. There are no errors in the chrome console or messages on the screen.

**Comment by cedrozor** [comment]
So, did you tried to connect with the IP address instead of hostname?

**Comment by cedrozor** [comment]
Possibly fixed into [myrtille version 1.7.0](https://github.com/cedrozor/myrtille/releases/tag/v1.7.0).

 If it doesn't, could you enable the logs by setting the following config key to "true" (into https://github.com/cedrozor/myrtille/blob/master/Myrtille.Services/app.config#L93).

```
  <appSettings>
    <!-- log remote session events into a file (located into <Myrtille folder>\log) -->
    <add key="RemoteSessionLog" value="true" />
  </appSettings>
```

 Then restart the Myrtille.Services (windows service).

**Comment by Arcath** [comment]
Yes sorry, when using IP it works fine.
